package.json

```diff
@@ -5,6 +5,8 @@
   "description": "",
   "scripts": {
     "build:examples": "pnpm --no-bail --workspace-concurrency 1 --filter '@example/*' build",
+    "lint": "eslint .",
+    "lint:ci": "eslint . --max-warnings 0",
```

Comment:
Not sure why we may need 2 commands? I think we could only get away with `eslint . --max-warnings 0` and if a user explicitly runs it locally, he gets the same result as in CI. If I am not missing anything, the only reason to have 2 commands could be if we had git hooks and we would not want to prevent users from pushing non valid code?